ansible/README.md

```diff
@@ -43,5 +43,15 @@ to enter the SSH password:
 The `--tags` option tells Ansible to only execute tasks with the specified tag.
 In the previous examples, only tasks with the `setup` tag would be executed.
 
-Provide the data requested by the playbook (eg Jenkins admin user name/password
-and SSH keys locations) and wait automatic setup finish.
+For Jenkins master playbook you only need to provide a jenkins username and password in 
+the target host.
+
+Jenkins slave playbook will prompt you to provide the following data:
+- Jenkins private SSH key file path _(press Enter to keep default value)_
+- Jenkins public SSH key file path _(press Enter to keep default value)_
+- Upload server user's private SSH key file path _(enter your SSH private key)_
+- GitHub user's private SSH key file path _(enter your SSH private key)_
+- Path to file containing known keys for GitHub and upload server hosts _(press Enter to keep 
+default known_hosts file path or enter any alternate file you may have)_
```

Comment:
I don't think the comments in parentheses help much. I'd explain better what the keys are used for:
Master:
- Jenkins admin user name and password: used to access the Jenkins UI with administrative privileges.
- Jenkins private and public SSH key file path: used to communicate between the master and the slave nodes.

Slave:
- Jenkins private and public SSH key file path: used to communicate between the master and the slave nodes.
- Upload server user's private SSH key file path: used to upload builds to the configured server.
- GitHub user's private SSH key file path: used to push commits to GitHub in periodic pipelines.
- Path to file containing known keys for GitHub and upload server hosts: used to recognize the server identities.

Reply:
It is certainly good to explain what each key means, but that still doesn't explain which ones are automatically generated and which ones are user generated (therefore with a pre-existing path provided by the user).